docs/users-manual/job-scheduling.rst

```diff
@@ -1,11 +1,148 @@
+Job Scheduling
+==============
+
+Priorities and Preemption
+-------------------------
+
+HTCondor has two independent priority controls: job priorities and user
+priorities.
+
+The HTCondor system calculate a "fair share" of machine slots to allocate to each user.
+Whether each user can use all of these slots depends on a number of factors. For example,
+if the user's jobs only match to a small number of machines, perhaps
+the user will be running fewer jobs than allocated.  This fair share is based on the
+*user priority*.  Each user can then specify the order in which each of their jobs
+should be matched and run on the fair share, this is based on the *job priority*.
+
+.. _jobprio:
+
+Job Priority
+''''''''''''
+
+:index:`priority<single: priority; job>` :index:`of a job<single: of a job; priority>`
+
+Job priorities allow a user to sort their own jobs to determine which are
+tried to be run first.  A job priority can be any integer: larger values 
+denote better priority.  So, 0 is a better job priority than -3, and 6 is a better than 5.
+:index:`condor_prio<single: condor_prio; HTCondor commands>`
```

Comment:
This index seems out of place currently. Should we remove it or find a way to better incorporate this via :tool:`condor_prio`?

Reply:
Just noticed that we use `:tool:'condor_prio'` later in this section so I think this index should be removed

Reply:
Pushed fix.